src/Components/Alert/AlertProvider.tsx

```diff
@@ -115,26 +118,33 @@ export const AlertProvider: FC<AlertProviderProps> = ({
 
   const handleComplete = async () => {
     try {
-      const reponse = await submitMutation({
+      const reponse = await submitCreateAlert({
         variables: {
           input: {
-            attributes: state.criteria,
-            userAlertSettings: state.settings,
-          },
+            ...state.criteria,
+            settings: state.settings,
+          } as { artistIDs: string[] }, // artistIDs is required in the input type
         },
       })
 
+      const alertID = reponse.createAlert?.responseOrError?.alert?.internalID
       const searchCriteriaID =
-        reponse.createSavedSearch?.savedSearchOrErrors.internalID
+        reponse.createAlert?.responseOrError?.alert?.searchCriteriaID
+      if (alertID) {
+        dispatch({
+          type: "SET_ALERT_ID",
+          payload: alertID,
+        })
+      }
 
       if (searchCriteriaID) {
```

Comment:
We keep this around b/c it is read out of this state and injected into an analytics tracking payload, confirmed in Slack that we do want to keep this `search_criteria_id` value in that payload for now.